UnityProject/Assets/Scripts/Doors/DoorController.cs

```diff
@@ -249,15 +249,17 @@ public void PlayOpenSound()
 		if (openSFX != null)
 		{
 			// Need to play this sound as global - this will ignore muffle effect
-			SoundManager.PlayAtPosition(openSFX, registerTile.WorldPosition, gameObject, polyphonic: true, isGlobal: true);
+			// JESTER
+			//SoundManager.PlayAtPosition(openSFX, registerTile.WorldPosition, gameObject, polyphonic: true, isGlobal: true);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)